docs/01GettingStarted/02-Biosensing-Setups/03-ECG-Setup.md

```diff
@@ -94,7 +94,7 @@ Once you have the GUI open, turn off all channels that are not connected to elec
 
 Once your settings have been adjusted, press 'Start Data Stream’.' You should see graphs similar to the ones below.
```

Comment:
We have double quotes here. Can this be fixed?

Reply:
Fixed!